Empire/Empire.cpp

```diff
@@ -347,14 +348,17 @@ namespace {
             // record allocation from group
             float group_drawdown = std::min(allocation, group_pp_available);
             allocated_pp[group] += group_drawdown;  // relies on default initial mapped value of 0.0f
+            if (queue_element.item.build_type == BT_STOCKPILE) {
+                stockpile_transfer += group_drawdown;
+            }
             group_pp_available -= group_drawdown;
 
-            float stockpile_drawdown = allocation <= group_drawdown ?  0.0f : (allocation - group_drawdown);
-                                       // 0.0f : (allocation - group_drawdown) / stockpile_conversion_rate;
+            float stockpile_drawdown = allocation <= group_drawdown ? 0.0f : (allocation - group_drawdown);
```

Comment:
Although it does look like you default the stockpile item to *not* draw from the stockpile, I don't see anything preventing the user from carelessly enabling it, which could easily lead to the stockpile item actually drawing down from the stockpile here, causing the troubles which I clarified in the discussion with my previous comment.   One way to change that could be where I previously proposed, another could be to disable the stockpile item from having stockpile-drawdown ever being enabled, or some adjustment could be made here.

I suppose the best thing would be to prevent the stockpile item from ever being enabled to drawdown from the stockpile, but that would need a bit of UI work.  In the meantime, one of these other two places could at least fix it to not do any harm.

Reply:
Oh. Thats what you meant. Best thing is to not show the enable stockpile option to the user (because it doesnt make sense), I'll try to implement that.

Reply:
Done